### PR TITLE
online banking poland for V4

### PR DESCRIPTION
**Pull request by etman55**

### CODE REVIEW

File: drop-in/build.gradle [hunk 1]
```diff
@@ -71,6 +71,7 @@ dependencies {
     api project(':bacs')
     api project(':wechatpay')
     api project(':voucher')
+    api project(':online-banking-pl')
```
Comment (jreij) [inline drop-in/build.gradle:74]: Alphabetical order

File: online-banking-pl/build.gradle [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+    defaultConfig {
+        minSdkVersion min_sdk_version
+        targetSdkVersion target_sdk_version
+        versionCode version_code
+        versionName version_name
+
+        testInstrumentationRunner 'androidx.test.runner.AndroidJUnitRunner'
+        consumerProguardFiles "consumer-rules.pro"
+    }
```
Comment (jreij) [inline online-banking-pl/build.gradle:31]: Don't we need this here?

```groovy


    compileOptions {
        sourceCompatibility JavaVersion.VERSION_1_8
        targetCompatibility JavaVersion.VERSION_1_8
    }

    kotlinOptions {
        jvmTarget = JavaVersion.VERSION_1_8.toString()
    }
```

**Comment by jreij** [comment]
You missed `PaymentMethodDetails.getChildSerializer`

**Comment by jreij** [comment]
Did you test the app? It crashes when you click on Online Banking PL

File: online-banking-pl/src/main/java/com/adyen/checkout/onlinebankingpl/OnlineBankingPLSpinnerView.kt [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package com.adyen.checkout.onlinebankingpl
```
Comment (jreij) [inline online-banking-pl/src/main/java/com/adyen/checkout/onlinebankingpl/OnlineBankingPLSpinnerView.kt:1]: Missing copyright

File: drop-in/build.gradle (hunk 1 above)
Comment (etman55) [reply drop-in/build.gradle:74]: > Did you test the app? It crashes when you click on Online Banking PL

what's the crash that you get? It doesn't crash with me!

**Comment by etman55** [comment]
> Did you test the app? It crashes when you click on Online Banking PL

what's the crash that you get? It doesn't crash with me!

**Comment by jreij** [comment]
@etman55 
```
2022-09-22 13:03:27.398 13662-13662/com.adyen.checkout.example E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.adyen.checkout.example, PID: 13662
    kotlin.UninitializedPropertyAccessException: lateinit property component has not been initialized
        at com.adyen.checkout.dropin.ui.base.BaseComponentDialogFragment.getComponent(BaseComponentDialogFragment.kt:48)
        at com.adyen.checkout.dropin.ui.component.GenericComponentDialogFragment.onViewCreated(GenericComponentDialogFragment.kt:70)
```

**Review by jreij — APPROVED** [review]

**Pull request merged**